system-setup.py

```diff
@@ -22,10 +22,14 @@ def common_first(self):
         self.install("git jq curl")
         self.run("%s/bin/enable-utf8" % READIES)
 
-        if self.os != 'macos' and self.dist != 'centos':
+        self.install("autoconf libtool")
+
+        if self.dist == 'fedora':
```

Comment:
This isn't accurage. On redhat based distros it's openssl-devel, on osx it's openssl, and on debiat distros it's libssl-dev.

You should use the per-distro function (is redhat_compat, debian_compat, etc).